index.html

```diff
@@ -18,63 +18,73 @@
-        <ul class="social-media">
-          <li>
-            <a href="#"
-              ><img
-                src="/img/icons/twitter.svg"
-                alt="twitter"
-                title="follow me on twitter"
-            /></a>
-          </li>
-          <li>
-            <a href="#"
-              ><img
-                src="/img/icons/Linkedin_icon.svg"
-                alt="linkedin"
-                title="follow me on linkedin"
-            /></a>
-          </li>
-          <li>
-            <a href="#"
-              ><img
-                src="/img/icons/medium.svg"
-                alt="medium"
-                title="follow me on"
-            /></a>
-          </li>
-          <li>
-            <a href="#"
-              ><img
-                src="/img/icons/github.svg"
-                alt="github"
-                title="follow me on github"
-            /></a>
-          </li>
-          <li>
-            <a href="#"
-              ><img
-                src="/img/icons/angellist.svg"
-                alt="angellist"
-                title="follow me on"
-            /></a>
-          </li>
-        </ul>
+        <div class="showcase__presentation">
+          <h1>I’m Joseph Glad to see you!</h1>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contactnxn me.
+          </p>
+
+          <a href="#"><p class="follow">let's connect</p></a>
+          <ul class="social-media">
+            <li>
+              <a href="#"
+                ><img
+                  src="/img/icons/twitter.svg"
+                  alt="twitter"
+                  title="follow me on twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="/img/icons/Linkedin_icon.svg"
+                  alt="linkedin"
+                  title="follow me on linkedin"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="/img/icons/medium.svg"
+                  alt="medium"
+                  title="follow me on"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="/img/icons/github.svg"
+                  alt="github"
+                  title="follow me on github"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="/img/icons/angellist.svg"
+                  alt="angellist"
+                  title="follow me on"
+              /></a>
+            </li>
+          </ul>
+        </div>
       </section>
       <section class="works">
         <div class="works__image">
```

Comment:
- [ ] In works section 2 of your cards are more height than the others. I would recommend fixing this just removing `height` property from ` .works `class 👍

**Your design**
![image](https://user-images.githubusercontent.com/69011963/149137992-e714629a-6500-46cd-bf12-49f19c1a431c.png)

**Figma design**
![image](https://user-images.githubusercontent.com/69011963/149138098-287ff677-8e6e-4a5c-a2b2-830a51af874b.png)

- [ ] You have used section tags correctly in other sections like about and hero, why not also use them in also works section 👌.  I would suggest you wrap all the works cards in one section, instead of using a section for all of them. You can change section tags to divs and wrap them in one section 😄👍

**Your code from dev tools 🔍 **
![image](https://user-images.githubusercontent.com/69011963/149138841-b4383d40-c97d-495f-be9a-abb7f5cc41da.png)